test/powershell/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+Describe "Native Command Arguments" -tags "CI" {
+    $testPath = 'TestDrive:\test.txt'
+    $powershellTestDir = $PSScriptRoot
+    while ($powershellTestDir -notmatch 'test[\\/]powershell$') {
+        $powershellTestDir = Split-Path $powershellTestDir
+    }
+    $echoArgs = Join-Path (Split-Path $powershellTestDir) tools/EchoArgs/bin/echoargs
+
+    It "Should handle quoted spaces correctly" {
+        $a = 'a"b c"d'
+        & $echoArgs $a 'a"b c"d' a"b c"d >$testPath
+        $testPath | Should Contain 'Arg 0 is <ab cd>'
+        $testPath | Should Contain 'Arg 1 is <ab cd>'
+        $testPath | Should Contain 'Arg 2 is <ab cd>'
```

Comment:
I was trying to do what I thought would be reasonable, but it didn't seem to work with Pester:

```powershell
& $echoArgs $a 'a"b c"d' a"b c"d | Should Match @"
Arg 0 is <ab cd>
Arg 1 is <ab cd>
Arg 2 is <ab cd>
"@
```

It seems like Pester only tries to match the first line of output when using `Should Match`. 😕